gradle.properties

```diff
@@ -6,7 +6,7 @@
 # http://www.gradle.org/docs/current/userguide/build_environment.html
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
-org.gradle.jvmargs=-Xmx1536m
+org.gradle.jvmargs=-Xmx1024m
```

Comment:
unexpected addition. Remove it.

Reply:
@rob729 this PR was made before I read your  comment regarding delete all profiles. Please don't waste time over this. I'll comment when I resolve all the bugs and push the code